examples/full_example/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ publish_to: none
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.18.2 <3.0.0"
+  sdk: '>=3.0.0 <4.0.0'
```

Comment:
Let's not do any edits to this for now,
I will do a version pump to all examples later.

Reply:
done